### ISSUE TITLE
[BUG] JikanTS baseUrl not including '/v3/' when performing search function

### ISSUE THREAD

**Issue by MrShoopa** [Julien-Broyard/JikanTS#187]
**Describe the bug**
I've been attempting to fetch Anime info using the default ```search``` function, and even when putting in correct parameters and testing it on RunKit, the results come as undefined. I went to see what the issue could be and it turns out it was using a malformed base URL.

**To Reproduce**
Perform any ```.search``` function.

**Expected behavior**
A result with the list of search results from MyAnimeList

**Version**
1.2.13

**Screenshots**

![image](https://user-images.githubusercontent.com/6378303/80254163-88819500-862f-11ea-8384-b6a4a05c4687.png)

![image](https://user-images.githubusercontent.com/6378303/80254059-53754280-862f-11ea-9231-dfdd4dab1198.png)

**Additional context**

**Comment by Julien-Broyard** [comment]
Hey, thanks for taking the time to report the bug. I'm actively working on the V2.0.0 which lies here : https://github.com/Julien-Broyard/jikan-ts

The version 2.0.0 solves a lot of the problem with the current version. I won't fix this issue in this version because I made this lib when I had poor skills in typescript. However I'll make sure to make the Search work (I have added unit tests | CI/CD) so it should be a better project for everyone to use and collaborate on

:+1:
